src/controllers/hid/hidcontroller.cpp

```diff
@@ -63,13 +63,12 @@ HidController::HidController(const hid_device_info& deviceInfo, UserSettingsPoin
     // which is which
     if (hid_interface_number < 0) {
         setDeviceName(
-            QString("%1 %2").arg(hid_product)
-            .arg(hid_serial.right(4)));
+                QString("%1 %2").arg(hid_product, hid_serial.right(4)));
     } else {
-        setDeviceName(
-            QString("%1 %2_%3").arg(hid_product)
-            .arg(hid_serial.right(4))
-            .arg(QString::number(hid_interface_number)));
+        setDeviceName(QString("%1 %2_%3")
```

Comment:
Will conflict with #3308, but don't mind. Moved into a separate class and replaced by QStringBuilder.